packages/@adobe/spectrum-css-temp/components/dialog/index.css

```diff
@@ -441,6 +441,9 @@ governing permissions and limitations under the License.
     border-top-right-radius: var(--spectrum-dialog-border-radius);
     overflow-y: auto;
 
+    /* Account for border-size so that the scrolling grid does not overflow the Dialog. */
+    margin-bottom: calc(2 * var(--spectrum-popover-border-size));
```

Comment:
Without this change, the scrolling content extends beyond the bounds of the dialog:
<img width="423" alt="Screenshot showing popover without this change showing content overflow" src="https://user-images.githubusercontent.com/154077/228000031-1660353f-320b-4ffe-a047-f785a27ce8a3.png">

Reply:
With the change, the scrolling content no longer extends beyond the bounds of the dialog:
<img width="423" alt="Screenshot demonstrating the fix so that the scrolling content no longer extends beyond the bounds of the dialog" src="https://user-images.githubusercontent.com/154077/228001235-eff6fa8a-b822-4d3f-a1bd-10f57d81e4e7.png">

Reply:
This seem specific to `.spectrum-Popover .spectrum-Dialog`, because `.spectrum-Popover` has a border that affects the height calculation of the  `.spectrum-Dialog` within.